README.md

```diff
@@ -227,11 +227,15 @@ docker run \
 > * The Pravega storage doesn't support reading the stream events in the random order.
 > * Works synchronously
 
-There is also another option, called `storage-driver-read-timeoutMillis`. Pravega documentation says it only works when
+There is a specific option (config parameter) for reading called `storage-driver-event-timeoutMillis`. Pravega documentation says it only works when
```

Comment:
recycle mode notes, example

Reply:
done

Reply:
example command to run the events reading is also necessary